### ISSUE TITLE
TEST-67-INTEGRITY is flaky

### ISSUE THREAD

**Issue by bluca** [systemd/systemd#35363]
```
[   19.432622] TEST-67-INTEGRITY.sh[649]: + '[' -e /dev/mapper/integrity_test ']'
[   19.442957] TEST-67-INTEGRITY.sh[649]: + '[' 1 -eq 1 ']'
[   19.445938] TEST-67-INTEGRITY.sh[745]: ++ integritysetup status integrity_test
[   19.448938] TEST-67-INTEGRITY.sh[747]: ++ awk '{print $2}'
[   19.450899] TEST-67-INTEGRITY.sh[746]: ++ grep '^\s*device:'
[   19.491588] TEST-67-INTEGRITY.sh[649]: + dev_name=/dev/loop1
[   19.492338] TEST-67-INTEGRITY.sh[748]: ++ blkid -U 01234567-ffff-eeee-eeee-0123456789ab
[   19.505748] TEST-67-INTEGRITY.sh[649]: + '[' /dev/loop1 '!=' /dev/mapper/integrity_test ']'
[   19.511679] TEST-67-INTEGRITY.sh[649]: + echo 'Failed to locate FS with matching UUID!'
[   19.511970] TEST-67-INTEGRITY.sh[649]: Failed to locate FS with matching UUID!
```

https://github.com/systemd/systemd/actions/runs/12028557879/job/33531956456?pr=35353

[TEST-67-INTEGRITY.journal.gz](https://github.com/user-attachments/files/17920056/TEST-67-INTEGRITY.journal.gz)

**Comment by yuwata** [comment]
BTW, the expected device node symlink is wrong. Comprehensive fix is waiting in #34505.
But, I will try to tape over the issue.

**Comment by yuwata** [comment]
-> #35366
